rwgame/RWGame.cpp

```diff
@@ -108,6 +108,9 @@ RWGame::RWGame(Logger& log, int argc, char* argv[])
         }
     });
 
+    setHUDScale(config.getHUDScale());
```

Comment:
Can code of these lines be located closer to comment `Set up text renderer`? For example after line 77 or 82.